src/main/java/nus/climods/model/module/UserModule.java

```diff
@@ -0,0 +1,140 @@
+package nus.climods.model.module;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import org.openapitools.client.api.ModulesApi;
+
+import nus.climods.logic.commands.exceptions.CommandException;
+import nus.climods.logic.parser.exceptions.ParseException;
+import nus.climods.model.Model;
+
+/**
+ * Class representing module a User has in their My Modules list
+ */
+public class UserModule {
+    public static final String MESSAGE_MODULE_NOT_FOUND = "Module not in current NUS curriculum";
+    private final ModulesApi api = new ModulesApi();
+
+    // Identity fields
+    private final Module apiModule;
+    private final String acadYear = "2022-2023";
+    private String tutorial = "Tutorial: Monday, 1400-1500";
```

Comment:
nit: Add a `TODO` or a issue to remove this in the future when we implement tut/lecture support